### ISSUE TITLE
Different results between NameChecker und PositionConverter

### ISSUE THREAD

**Issue by DanielaDey** [mutalyzer/mutalyzer#471]
There are two problems here that exist on both the website and the SOAP:

1: Using the NameChecker without a NM-number version number tells me the newest version number, which I then use to convert to a genomic reference. Here the newest version number can sometimes not be found when using the PositionConverter. Example: NM_024408.4(NOTCH2):c.545A>T

2. The NameChecker tells me my input is correct, but no the position conversion fails. Example: NM_001346765.1(MYO18A):c.3590C>T

**Comment by mihailefter** [comment]
Dear Daniela, as far as we know at the moment NCBI did not release any new annotation set. So we cannot retrieve any information about `NM_024408.4` in `NC_000001.11`, in which there is only data about `NM_024408.3`.

For `NM_001346765.1` it is similar. The Name Checker works in this case because it does not require any NC information.

Please note that, even with a new NCBI annotation release, there will be some delay until Mutalyzer will be aware of this.

**Comment by DanielaDey** [comment]
Thank you for you explanation. Sadly this makes the usage very hard for my purpose. 
Is there anyway I could get the error messages that I get on the web version also using the SOAP client? The webversion tells me, which version numbers are known, when I try to convert using the newest number which is not in the annotation set. The SOAP client only returns None whenever its unable to convert. If it gave me the error message I could retry converting using the given version number.

**Comment by mihailefter** [comment]
Could you please provide an example where such a difference occurs?

**Comment by DanielaDey** [comment]
If I put NM_024408.4(NOTCH2):c.545A>T in the position convert on the website, it get notified, that it doesn't know version 4, but version 2 and 3. With this information I can then of course change my input. 

Using the SOAP client in python, I only get [None] back for this input using the following code:

```python
import zeep

client = zeep.Client('https://mutalyzer.nl/services/?wsdl')
var_g = client.service.numberConversion('hg19', 'NM_024408.4(NOTCH2):c.545A>T')

print(var_g)
```

**Comment by mihailefter** [comment]
Indeed, the webservice "equivalent" of the Position Converter does not provide the same feedback as the browser version. We will take this into consideration for future releases.

The `runMutalyzerLight` webservice endpoint however, provides some more information under `messages` section:
https://mutalyzer.nl/json/runMutalyzerLight?variant=NC_000001.11(NM_024408.4):c.545A%3ET

**Comment by DanielaDey** [comment]
Thank you, getting a proper error message would make it a lot easier to handle problems.
